lib/blockscore/response.rb

```diff
@@ -16,13 +16,13 @@ def handle_response(resource, response)
     def api_error(response)
       case response.code
       when 400
-        fail InvalidRequestError.new(response)
+        raise InvalidRequestError.new(response)
```

Comment:
Maybe you also want to adopt the `fail` convention when raising "new" exceptions.